site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,233 @@
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close pane."
+
+___
+
+### TabController
+
+TabController creates a set of tabs that control the content of a container. The tabs use `role="tab"` and are associated with the content through `aria-controls` and `aria-labelledby`. Only one tab is in the tab order, and arrow keys are used to switch between tabs.
+
+##### Keyboard Interaction
+- **Left arrow**: Moves to previous tab, wrapping from first to last
+- **Right arrow**: Moves to next tab, wrapping from last to first
+- **Down arrow**: Moves to next tab on vertically oriented tabs
+- **Up arrow**: Moves to previous tab on vertically oriented tabs
+- **Page Up**: Moves to first tab
+- **Page Down**: Moves to last tab
+- **Escape**: Closes focused tab if it is closeable
+
+___
+
+### TitlePane
+
+TitlePane creates an accordion widget, functionally a button that expands a content container. Collapsed content is obscured with `aria-hidden`, and the button and content container are associated with `aria-controls` and `aria-labelledby`.
+
+##### A11y properties
+- `headingLevel`: Optionally customize the heading level of the button controlling the accordion.
+___
+
+This overview only touched on properties that primarily exist for accessibility-related reasons, but any design decision or property change will end up affecting accessibility in some way. For some of those, such as `invalid`, `disabled`, and `readOnly`, we handle setting ARIA attributes in the background in addition to toggling classes without any extra attention needed from the user. In other cases, such as `getResultLabel` in ComboBox or `renderMonthLabel` in Calendar, it is entirely up to the widget user to ensure the returned result includes clear, screen-reader-accessible text content.
+
+## Styling
+The base Dojo 2 theme meets WCAG AA color contrast guidelines, but it is up to the user to double check the final styles as changes to font size or background could affect readability. Widget css packaged with Dojo2 also includes `:focus` styles for all focusable nodes.
+
+There is a set of base styles provided by `@dojo/widgets` separate from themes, containing basic utility classes like the screen reader-accessible visually hidden style. To use it, import `baseCss` separately like so:
+
+```js
+import * as css from ‘path/to/your/css’;
+import * as baseCss from ‘@dojo/widgets/common/styles/base.m.css’;
+
+@theme(css)
+class MyWidget extends WidgetBase {
+	render() {
+		return v(‘div’, {
+			classes: this.classes().fixed(baseCss.visuallyHidden)
+		}, [ ‘Screen reader instructions’ ]);
+	}
+}
+```
+
+## Focus Management
+Dojo 2 will provide a focus manager for situations where focus needs to be directly managed, since directly touching the DOM within widgets is discouraged. This feature is currently in development.
```

Comment:
Do we have any articles that could be linked here that talk about the reasons why manipulating the DOM is discouraged?